test/test_json.py

```diff
@@ -18,28 +19,35 @@
     from urllib.request import urlopen
 
 NVD_SCHEMA = "https://scap.nist.gov/schema/nvd/feed/1.1/nvd_cve_feed_json_1.1.schema"
-NVD_2019_META = "https://nvd.nist.gov/feeds/json/cve/1.1/nvdcve-1.1-2019.meta"
+NVD_2019_META = "https://nvd.nist.gov/feeds/json/cve/1.1/nvdcve-1.1-2020.meta"
```

Comment:
We should probably change this to `NVD_META` or something.

Reply:
Hm, we're not even using this any more; I might as well remove it.